From 2793a31faed8700584b964c7caf3b027d1b326b5 Mon Sep 17 00:00:00 2001
From: Calin Juravle <calin@google.com>
Date: Mon, 12 Apr 2021 22:03:29 -0700
Subject: [PATCH] Fix dm-test invocation

Due to a bad merge --copy-dex-files=false was passed to the vdex_cmdline
instead of the dex2oat_cmdline.

Test: test-art-host
Bug: 178055795

Merged-In: I5b104329af899ea62dc1779800264bad6b5424a4
Change-Id: Ieb6e324232996ec8ad5119fcbf20b81d62678fd2
(cherry picked from commit 6c9431c02e739de83c7c804d64e4a4656d24d55f)
---
 test/etc/run-test-jar | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/etc/run-test-jar b/test/etc/run-test-jar
index 423f9b3c01..7d85b102b5 100755
--- a/test/etc/run-test-jar
+++ b/test/etc/run-test-jar
@@ -963,9 +963,9 @@ if [ "$PREBUILD" = "y" ]; then
   elif [ "$TEST_VDEX" = "y" ]; then
     vdex_cmdline="${dex2oat_cmdline} ${VDEX_ARGS} --input-vdex=$DEX_LOCATION/oat/$ISA/$TEST_NAME.vdex"
   elif [ "$TEST_DM" = "y" ]; then
-    dex2oat_cmdline="${dex2oat_cmdline} --output-vdex=$DEX_LOCATION/oat/$ISA/primary.vdex"
+    dex2oat_cmdline="${dex2oat_cmdline} --copy-dex-files=false --output-vdex=$DEX_LOCATION/oat/$ISA/primary.vdex"
     dm_cmdline="zip -qj $DEX_LOCATION/oat/$ISA/$TEST_NAME.dm $DEX_LOCATION/oat/$ISA/primary.vdex"
-    vdex_cmdline="${dex2oat_cmdline} ${VDEX_ARGS} --copy-dex-files=false --dump-timings --dm-file=$DEX_LOCATION/oat/$ISA/$TEST_NAME.dm"
+    vdex_cmdline="${dex2oat_cmdline} ${VDEX_ARGS} --dump-timings --dm-file=$DEX_LOCATION/oat/$ISA/$TEST_NAME.dm"
   elif [ "$PROFILE" = "y" ] || [ "$RANDOM_PROFILE" = "y" ]; then
     vdex_cmdline="${dex2oat_cmdline} --input-vdex=$DEX_LOCATION/oat/$ISA/$TEST_NAME.vdex --output-vdex=$DEX_LOCATION/oat/$ISA/$TEST_NAME.vdex"
   fi
--
2.29.0
